### PR TITLE
PIM-4129: css new style stephane

### PR DESCRIPTION
**Pull request by yulldesign**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/AttributeGroup/form.html.twig [hunk 1]
```diff
@@ -25,7 +25,7 @@
                 'flash.attribute group.removed'|trans)
             }}
         {% endif %}
-        {{ elements.submitBtn() }}
+        {{ elements.submitBtn('Save', 'ok') }}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/AttributeGroup/form.html.twig]: You should use translation keys

File: src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig [hunk 2]
```diff
@@ -55,7 +55,7 @@
         {% elseif parentId %}
             {{ elements.backLink(path('pim_enrich_categorytree_edit', { id: parentId }), 'btn.to parent') }}
         {% endif %}
-        {{ elements.submitBtn() }}
+        {{ elements.submitBtn('Save', 'ok') }}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig]: please use translation keys

File: src/Pim/Bundle/EnrichBundle/Resources/views/Group/_group.html.twig [hunk 3]
```diff
@@ -24,7 +24,7 @@
                 confirmationMessage|trans({'%name%': form.vars.value.label}),
                 removeMessage|trans
             ) }}
-            {{ elements.submitBtn() }}
+            {{ elements.submitBtn('Save', 'ok') }}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Group/_group.html.twig]: translation key

File: src/Pim/Bundle/EnrichBundle/Resources/views/VariantGroup/edit.html.twig [hunk 4]
```diff
@@ -42,7 +42,7 @@
                 'confirmation.remove.variant group'|trans({'%name%': form.vars.value.label}),
                 'flash.variant group.removed'|trans
             ) }}
-            {{ elements.submitBtn() }}
+            {{ elements.submitBtn('Save', 'ok') }}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/VariantGroup/edit.html.twig]: translation key

File: src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/edit.html.twig [hunk 5]
```diff
@@ -23,7 +23,7 @@
         {% set buttons %}
             {{ elements.backLink(indexRoute) }}
             {{ elements.deleteLink(removeRoute, removeAcl, indexRoute, removeMessage, removeSuccessMessage) }}
-            {{ elements.submitBtn }}
+            {{ elements.submitBtn('Save', 'ok') }}
```
Comment (sumbobyboys) [inline src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/edit.html.twig]: translation key

File: src/Pim/Bundle/UIBundle/Resources/public/css/accordion.less [hunk 6]
```diff
@@ -0,0 +1,20 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+.accordion {
+  margin-bottom: 0;
+
+  .accordion-group {
+    margin: 0;
+    border-radius: 0px;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/accordion.less]: 0

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 7]
```diff
@@ -0,0 +1,331 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+.navbar-extra .btn {
+  margin-left: 4px;
+}
+
+.btn{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: please add a blank line before {

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+  .transition();
+
+  &:hover, &:active, &:focus{
+    background-image: none;
+    border: none;
+    box-shadow: none;
+    background-color: darken(#E7E7EC, 10%);
+  }
+}
+.btn.btn-primary {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: please, apply code style before pushing anything :)

Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: in this case, add a blank line before

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+      line-height: 34px;
+      background-color: darken(@color-new-green, 10%);
+      margin: 0;
+    }
+  }
+
+  &:hover {
+    background-color: darken(@color-new-green, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 10]
```diff
@@ -0,0 +1,331 @@
...
+}
+
+.btn.btn-submit {
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 11]
```diff
@@ -0,0 +1,331 @@
...
+    height: 34px;
+    line-height: 34px;
+    background-color: darken(@color-new-green, 10%);
+    margin: 0;
+  }
+
+  &:hover {
+    background-color: darken(@color-new-green, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 12]
```diff
@@ -0,0 +1,331 @@
...
+    height: 34px;
+    line-height: 34px;
+    background-color: darken(@color-new-red, 10%);
+    margin: 0;
+  }
+
+  &:hover {
+    background-color: darken(@color-new-red, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank lone

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 13]
```diff
@@ -0,0 +1,331 @@
...
+    background-color: darken(@color-new-green, 10%);
+  }
+
+}
+
+.btn-publish {
+  background-color: #5992C7;
+
+  &:hover {
+    background-color: darken(#5992C7, 10%);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: maybe use a variable ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 14]
```diff
@@ -0,0 +1,331 @@
...
+  border-radius: 2px;
+
+  i {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 34px;
+    height: 34px;
+    line-height: 34px;
+    background-color: darken(#A1A9B7, 10%);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: maybe use a variable for #A1A9B7

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 15]
```diff
@@ -0,0 +1,331 @@
...
+
+  &:hover {
+    background-color: darken(#A1A9B7, 10%);
+  }
+
+}
+
+.btn.delete-product {
+  background-color: @color-new-red;
+  color: #FFF;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: use lower cases for colors (please apply this everywhere)

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 16]
```diff
@@ -0,0 +1,331 @@
...
+  padding: 0 10px 0 44px;
+  overflow: hidden;
+
+  i {
+    position: absolute;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: please create a mixin for i general properties (you almost duplicated all the code from 20 lines above)

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 17]
```diff
@@ -0,0 +1,331 @@
...
+    background-color: darken(@color-new-red, 10%);
+    margin: 0;
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 2px;
+  }
+
+  &:hover {
+    background-color: darken(@color-new-red, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 18]
```diff
@@ -0,0 +1,331 @@
...
+}
+
+.btn.cancel {
+  background-color: #E7E7EC;
+
+  &:hover {
+    background-color: darken(#E7E7EC, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 19]
```diff
@@ -0,0 +1,331 @@
...
+}
+
+.btn.back-link {
+  background-color: #E7E7EC;
+  width: 34px;
+
+  &:hover {
+    background-color: darken(#E7E7EC, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 20]
```diff
@@ -0,0 +1,331 @@
...
+}
+
+.btn.back-link {
+  background-color: #E7E7EC;
+  width: 34px;
+
+  &:hover {
+    background-color: darken(#E7E7EC, 10%);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: use a variable

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 21]
```diff
@@ -0,0 +1,331 @@
...
+
+}
+
+.btn.pencil {
+  background-color: @color-new-blue;
+  color: #FFF;
+  padding: 0 10px 0 44px;
+
+  i {
+    position: absolute;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: same here, use a mixin

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 22]
```diff
@@ -0,0 +1,331 @@
...
+    height: 34px;
+    line-height: 34px;
+    background-color: darken(@color-new-blue, 10%);
+    margin: 0;
+  }
+
+  &:hover {
+    background-color: darken(@color-new-blue, 10%);
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 23]
```diff
@@ -0,0 +1,331 @@
...
+.btn-group > .btn:first-child {
+  border-radius: 2px;
+}
+
+.btn-group > .btn:last-child {
+  border-radius: 2px;
+}
+
+.filter-box {
+  .btn{
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: add space before {

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 24]
```diff
@@ -0,0 +1,331 @@
...
+    .transition();
+  
+    &:hover, &:active, &:focus {
+      background-image: none;
+      border: solid 1px #E7E7EC;
+      box-shadow: none;
+      background-color: darken(#FFF, 10%);
+    }
+  }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 25]
```diff
@@ -0,0 +1,331 @@
...
+  .attribute-copy-actions {
+    .btn {
+      background-color: #A1A9B7;
+      color: #FFF;
+
+      .caret {
+        border-top: 4px solid #FFF;
+      }
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 26]
```diff
@@ -0,0 +1,331 @@
...
+    margin-left: 0;
+  }
+
+  .pimmultiselect {
+    background-color: #FFF;
+  }
+
+  .selection-dropdown {
+    .btn {
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 27]
```diff
@@ -0,0 +1,73 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+.filter-box {
+  box-shadow: none;
+  background:#E7E7EC;
+  border:none;
+  padding: 15px 20px 8px 150px;
+
+  .icon-remove:before { color:#aaa; font-size:14px; }
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: nope

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 28]
```diff
@@ -0,0 +1,73 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+.filter-box {
+  box-shadow: none;
+  background:#E7E7EC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: please add a space before #E..

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 29]
```diff
@@ -0,0 +1,73 @@
...
+  border:none;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: same here

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 30]
```diff
@@ -0,0 +1,73 @@
...
+
+  .filter-item {
+
+    .filter-criteria-selector,
+    [class$=dropdown-toggle].btn {
+      height:29px;
+      line-height: 29px;
+      font-weight:normal;
+      padding-top:0px;
+      button {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: please add a blank line before ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 31]
```diff
@@ -0,0 +1,73 @@
...
+
+  .filter-list {
+    float: left;
+    margin: 0 0 0 -120px;
+    position: relative;
+  }
+
+  .filter-item {
+
+    .filter-criteria-selector,
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: nope, not on multiple lines :)

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+  .icon-remove:before { color:#aaa; font-size:14px; }
+
+  .filter-list {
+    float: left;
+    margin: 0 0 0 -120px;
+    position: relative;
+  }
+
+  .filter-item {
+
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: remove this blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 33]
```diff
@@ -0,0 +1,73 @@
...
+      }
+      .caret { margin-top:11px; }
+    }
+
+    .select-filter-widget .caret {
+      margin-top:8px; 
+    }
+
+    .choicefilter {
+      input[type='text'] {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: always double quotes in css, please fix everywhere

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 34]
```diff
@@ -0,0 +1,73 @@
...
+        margin-right:-7px;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: space

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 35]
```diff
@@ -0,0 +1,73 @@
...
+        height:29px;
+        line-height:29px;
+      }
+    }
+
+    .disable-filter {
+      line-height:18px !important;
+      background-color: transparent !important;
+      background-image: none;
+      border:none !important;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: space

File: src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less [hunk 36]
```diff
@@ -0,0 +1,73 @@
...
+        .border-radius(0 3px 3px 0);
+        position:relative;
+        font-size:14px;
+        height:29px;
+        line-height:29px;
+      }
+    }
+
+    .disable-filter {
+      line-height:18px !important;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less]: space

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 37]
```diff
@@ -785,16 +785,30 @@ div.dropdown-menu.channel {
...
     margin-bottom: 0;
     color: @textColor;
     cursor: pointer;
-    border: 1px solid #ccc;
-    border-radius: 4px;
+    border: none;
+    border-radius: 2px;
     box-shadow: none;
+    position: relative;
+    background-color: #E7E7EC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 38]
```diff
@@ -785,16 +785,30 @@ div.dropdown-menu.channel {
...
+    overflow: hidden;
     > i {
       font-size: 14px;
-      vertical-align: middle;
+      position: absolute;
+      top:0;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: space

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less [hunk 39]
```diff
@@ -444,7 +444,10 @@ select {
   margin-top: 10px;
   overflow: visible;
   .table-bordered {
-    border-left:1px solid #ddd;
+    border-left:1px solid #DFDFDF;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less [hunk 40]
```diff
@@ -464,16 +467,9 @@ select {
     width: 60px;
   }
   thead th {
-    background-color: #eee;
+    background-color: #E7E7EC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/oro.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less [hunk 41]
```diff
@@ -61,6 +61,10 @@
 @color-new-purple-dark: #5F2385;
 @color-new-purple-medium: #9452BA;
 @color-new-purple-light: #F2EBF8;
+@color-new-green: #33B5A1;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/less/variables.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 42]
```diff
@@ -7,6 +7,12 @@ body {
   color: @textColor;
   background-color: @backgroundColor-white;
 
+  &.login-page {
+    background-image: url("/bundles/pimui/images/background.svg");
+    background-size: cover;
+    background-color: #F1F4F9;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 43]
```diff
@@ -648,9 +654,9 @@ body {
 header.tab-header {
   height: 44px;
   border-left: none;
-  border-bottom: solid 1px #e5e5e5;
-  background: #f9f9f9;
-  padding: 0 20px;
+  border-bottom: solid 1px #CCC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 44]
```diff
@@ -722,7 +728,7 @@ h3.tab-header {
     max-width: @max-field-width + 10px;
     .flex(1);
     margin-left: 10px;
-    border-left: 1px solid #e5e5e5;
+    border-left: 1px solid #CCC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 45]
```diff
@@ -823,7 +828,7 @@ h3.tab-header {
         height: 44px;
         border-left: 1px solid #ddd;
         border-right: none;
-        background: #f9f9f9;
+        background: #E7E7EC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 46]
```diff
@@ -841,7 +846,7 @@ h3.tab-header {
   margin-top: 5px;
 
   .btn {
-    background: none;
+    background: #E7E7EC;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 47]
```diff
@@ -851,10 +856,10 @@ h3.tab-header {
     border-radius: 1px;
 
     &.active {
-      background: #555;
-      color: white;
+      background: darken(#E7E7EC, 10%);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 48]
```diff
@@ -1577,29 +1580,29 @@ h3.tab-header {
 .btn-dropdown {
   &.enabled > a, & > .enabled {
     opacity: 1;
-    background-color: #dae9de;
-    color: #67b373;
-    border: 1px solid #67b373;
+    background-color: darken(#A1A9B7, 10%);
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lower

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 49]
```diff
@@ -785,16 +785,30 @@ div.dropdown-menu.channel {
...
     cursor: pointer;
-    border: 1px solid #ccc;
-    border-radius: 4px;
+    border: none;
+    border-radius: 2px;
     box-shadow: none;
+    position: relative;
+    background-color: @color-new-grey;
+    overflow: hidden;
     > i {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: please add a & before

**Comment by phaseinducer** [comment]
recess warning :
>No need to specify units when a value is 0
>33. padding-top: 0px;

in src/Pim/Bundle/UIBundle/Resources/public/css/filterbox.less

**Pull request closed without merging**